qiskit/providers/aer/backends/aer_simulator.py

```diff
@@ -472,6 +472,9 @@ def _default_options(cls):
             accept_distributed_results=None,
             blocking_qubits=None,
             blocking_enable=False,
+            memory_blocking_bits=0,
```

Comment:
What does this do?

Reply:
memory_blocking_bits is an experimental flag so I removed from aer_simulator.py and this is disabled by default.

This option is used to enable GPU shared memory blocking to decrease memory access (GPU can simulate up to 10 qubits on shared memory without accessing main memory)